groups/models.py

```diff
@@ -161,6 +167,9 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def __str__(self):
+        return 'Comment by {}'.format(self.user.get_full_name())
```

Comment:
Information like which user made the comment can be shown by adding fields to the `list_display` on the admin class.